rpc/dial.go

```diff
@@ -295,11 +298,38 @@ func dial(
 	return conn, cached, nil
 }
 
+func listMulticastInterfaces() []net.Interface {
+	var interfaces []net.Interface
+	ifaces, err := net.Interfaces()
+	if err != nil {
+		return nil
+	}
+	for _, ifi := range ifaces {
+		if (ifi.Flags & net.FlagUp) == 0 {
```

Comment:
What is this checking for?

Reply:
if the interface is not up - the way flags are parsed and implemented is definitely a bit confusing

Reply:
Makes sense; mind leaving a comment above this line stating that?